backend/open_webui/models/prompts.py

```
@@ -118,17 +119,174 @@ def get_prompts(self) -> list[PromptUserResponse]:
+            # Apply pagination
+            offset = (page - 1) * limit
+            prompts = (
+                query.order_by(Prompt.timestamp.desc())
+                .offset(offset)
+                .limit(limit)
+                .all()
+            )
+
+            return [PromptModel.model_validate(prompt) for prompt in prompts]
+
+            return result
```

Ruff (0.12.2)

Undefined name `result`

(F821)